packages/browser/src/types/analytics.ts

```diff
@@ -26,41 +26,70 @@ export interface ExperimentAnalyticsEvent {
    * User properties to identify with the user prior to sending the event.
    */
   userProperties?: Record<string, unknown>;
```

Comment:
Since you added `userProperty` you dont need this anymore?

I am confused that we need both userProperty and userProperties.

Reply:
Yeah, this isn't necessary anymore but keeping it for backwards compatibility